server/src/test/scala/org/http4s/server/middleware/LoggerSpec.scala

```diff
@@ -27,10 +37,10 @@ class LoggerSpec extends Http4sSpec {
     }
 
     "not effect a Post" in {
-      val req = Request(uri = uri("/post"), method = POST).withBody(urlForm)
-      val res = req.flatMap(responseLoggerService.orNotFound)
+      val req = Request(uri = uri("/post"), method = POST).withBody(body)
```

Comment:
I'm getting sleepy and not sure why this is a `Request` instead of a `Task[Request]` like it was with `urlForm`.